grails-app/controllers/org/pih/warehouse/api/LocalizationApiController.groovy

```diff
@@ -58,7 +59,58 @@ class LocalizationApiController {
         Locale locale = localizationService.getLocale(languageCode)
         String message = messageSource.getMessage(params.id, params.list("args").toArray(), locale)
         render([code: params.id, message: message, currentLocale: locale] as JSON)
-
     }
 
+    static JSON localizeHelpScoutLabels(GrailsApplication app, Locale locale) {
```

Comment:
I couldn't get grails to recognize this method from a gsp without making it static.

Not wild about that; the preceding functions can call e.g. `localizationService.getCurrentLocale()`, but the caller has the info immediately at hand so it's not the end of the world. Still, I'm hoping there's a more grails-y way to do this.

Reply:
I'm confused. Why couldn't we use ${g.message()} in the GSP or even here?

Reply:
You can also use messageSource (see ApiController.getAppContext).